packages/network/lib/uri.ts

```diff
@@ -10,24 +10,24 @@ import url, { URL } from 'url'
 
 // yup, protocol contains a: ':' colon
 // at the end of it (-______________-)
-const DEFAULT_PROTOCOL_PORTS = {
+const DEFAULT_PROTOCOL_PORTS: Record<string, string> = {
```

Comment:
Seems this is never going to change - instead of  `Record<string, string>` we could do `as const` and get better type safety - that way, only `http` and `https` would be valid. `DEFAULT_PROTOCOL_PORTS.foo` would raise a typing error.

Reply:
Done.